src/webhook/__tests__/index.js

```diff
@@ -116,4 +124,56 @@ describe('Webhook router', () => {
       });
     });
   });
+
+  it('singleUserHandler() should reply default messages', async () => {
+    const userId = 'U4af4980630';
+    const app = new Koa();
+    app.use(webhookRouter.routes(), webhookRouter.allowedMethods());
+
+    const eventObject = {
+      events: [
+        {
+          replyToken: 'nHuyWiB7yP5Zw52FIkcQobQuGDXCTA',
+          type: 'messages',
+          mode: 'active',
+          timestamp: 1462629479859,
+          source: {
+            type: 'user',
+            userId,
+          },
+          message: {
+            id: '325708',
+            type: 'sticker',
+            packageId: '1',
+            stickerId: '1',
+            stickerResourceType: 'STATIC',
+          },
+        },
+      ],
+    };
+
+    const server = app.listen();
+
+    await request(server)
+      .post('/')
+      .send(eventObject)
+      .expect(200);
+
+    /**
+     * The HTTP response isn't guaranteed the event handling to be complete
+     */
+    await sleep(500);
+
+    // snapshot reply messages
+    expect(lineClient.mock.calls).toMatchSnapshot();
+    // snapshot user context
+    expect(redis.set.mock.calls).toMatchSnapshot();
```

Comment:
Nit: when invoking `toMatchSnapshot` multiple times in one `it` test, we can pass in snapshot names to make snapshots easier to read.